packages/odyssey-react-mui/src/labs/SideNav/SideNavItemContent.tsx

```diff
@@ -254,6 +254,7 @@ const SideNavItemContent = ({
             tabIndex={0}
             onClick={onClick}
             onKeyDown={sideNavItemContentKeyHandler}
+            role="button"
```

Comment:
If the content inside this container is a link, we shouldn't need this role. Or, any of these key/click handlers I wouldn't think

Reply:
I misunderstood what was happening here. What we'd actually need to do is render a semantic `button` element, rather than a div with a `role='button'` probably right?

Reply:
Changing this to a `button` breaks a bunch of styles; if we decide to do this, we should break it out into its own PR.